### ISSUE TITLE
Allow additional authentication parameters

### ISSUE THREAD

**Issue by hockdudu** [Drenso/symfony-oidc#20]
Some OIDC providers like Microsoft accept additional auth params like `login_hint` and `domain_hint`.[^microsoft-oidc]

It would be cool if there was an easy way to pass these parameters when generating the authorization request. Currently, it's only possible by modifying the redirect response.[^redirect-response]

This can be currently made in the Jumbojett's implementation by using the field `authParams`.[^jumbojett]

Maybe we could do this here with an additional parameter at `\Drenso\OidcBundle\OidcClient::generateAuthorizationRedirect` (e.g. `array $additionalParameters = []`)?

[^microsoft-oidc]: https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#send-the-sign-in-request
[^jumbojett]: See https://github.com/jumbojett/OpenID-Connect-PHP/blob/6aae75b1c7b58d8d64a6d14877f77b8a13ab90f6/src/OpenIDConnectClient.php#L594 and https://github.com/jumbojett/OpenID-Connect-PHP/blob/6aae75b1c7b58d8d64a6d14877f77b8a13ab90f6/src/OpenIDConnectClient.php#L797
[^redirect-response]:
    For instance:
    ```php
    $response = $oidcClient->generateAuthorizationRedirect();
    $targetUrl = $response->getTargetUrl();
    $targetUrl .= '&' . http_build_query(['domain_hint' => 'example.com']);
    return new RedirectResponse($targetUrl);
    ```

**Comment by bobvandevijver** [comment]
Thank you for the suggestion, you should feel free to open a pull request next time! Implemented now with 85b7eb9 and released with [v2.7.0](https://github.com/Drenso/symfony-oidc/releases/tag/v2.7.0).

Note that static parameters were already kept when set in the authorization endpoint itself.

**Comment by hockdudu** [comment]
Many thanks! :pray:
